libs/community/langchain_community/vectorstores/vectara.py

```diff
@@ -317,69 +363,101 @@ def add_texts(
             )
         return [doc_id]
 
-    def vectara_query(
+    def _get_query_body(
```

Comment:
thanks for underscore! Will prevent changes to this interface being breaking in future

Reply:
Not at all. That's why I realized it's best to rename as such.